modules/cnv-pxe-booting-with-mac-address.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv-configuring-pxe-booting.adoc
+
+[id="cnv-pxe-booting-with-mac-address-{context}"]
+= PXE booting with a specified MAC address
+
+PXE booting, or network booting, is supported in {ProductName}.
+Network booting allows a computer to boot and load an
+operating system or other program without requiring a locally attached
+storage device. For example, you can use it to choose your desired OS
+image from a PXE server when deploying a new host.
+
+.Prerequisites
+
+//note: the xref below needs to be changed after the CNV buckets are set up.
+
+* A Linux bridge must be connected, as described xref:../../cnv/cnv-attaching-vm-multiple-networks.adoc#connecting-resource-bridge-network-cnv-attaching-multiple-networks[here].
+* The PXE server must be connected to the same VLAN as the bridge.
+
+.Procedure
+
+. Configure a PXE network on the cluster:
+
+.. Create the NetworkAttachmentDefinition file for PXE network `pxe-net-conf`:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: pxe-net-conf
+spec:
+  config: '{
+      "cniVersion": "0.3.1",
+      "name": "pxe-net-conf",
+      "plugins": [
+        {
+          "type": "cnv-bridge",
+          "bridge": "br1",
+          "ipam": {}
+        },
+        {
+          "type": "cnv-tuning" <1>
+        }
+      ]
+    }'
+----
+<1> The `cnv-tuning` plug-in provides support for custom MAC addresses.
++
+[NOTE]
+====
+The VMI will be attached to the bridge `br1` through an access port with the
+requested VLAN.
+====
+
+. Edit the virtual machine instance configuration file to include the
+details of the interface and network.
+
+.. Specify the network and MAC address, if required by the PXE server.
+If the MAC address is not specified, a value is assigned
+automatically. However, note that at this time, MAC addresses
+assigned automatically are not persistent.
++
+Ensure that `bootOrder` is set to `1` so that the interface boots first.
+In this example, the interface is connected to a network called
+`<pxe-net>`:
++
+----
+interfaces:
+- masquerade: {}
+  name: default
+- bridge: {}
+  name: pxe-net
+  macAddress: de:00:00:00:00:de
+  bootOrder: 1
+----
++
+[NOTE]
+====
+Boot order is global for interfaces and disks.
+====
+
+.. Assign a boot device number to the disk to ensure proper booting
+after operating system provisioning.
++
+Set the disk `bootOrder` value to `2`:
++
+----
+devices:
+  disks:
+  - disk:
+      bus: virtio
+    name: containerdisk
+    bootOrder: 2
+----
+
+.. Specify that the network is connected to the previously created
```

Comment:
The part of creating the NetworkAttachmentDefinition is missing. The file was composed in line 25, but the NetworkAttachmentDefinition, which derives from this file, was never created.
oc create -f pxe-net-conf.yaml

Reply:
Aha! Excellent catch. I can add `oc create -f pxe-net-conf.yaml` after line 55, if that sounds good to you.

Reply:
Sounds like the perfect place for this addition.

Reply:
Added it in https://github.com/openshift/openshift-docs/pull/14351/commits/590c5e7c9d207998764b3ad260d8d3a2848da5ce

Reply:
Right in place, thanks!